onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -981,6 +995,13 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.input.shape) == 0,
         reason="fixme: Rank(0) input will lead ORT failed due to different rank(result) in if-else branch",
     ),
+    skip(
+        "scatter_reduce",
+        matcher=lambda sample: sample.kwargs.get("include_self") is False
+        or len(sample.input.shape)
+        == 0,  # rank 0 case failed even after converted to rank 1 tensor
```

Comment:
What was the failure? Was it value mismatch/ shape inference/ ORT crashes etc.?

Reply:
The reason should be: the graph is not correct, missing the logic under op.If().